### PR TITLE
USSD public - add servicerating flow

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
Add necessary flow/states to enable servicerating questions to be asked.
At start state check if identity exists and whether servicerating_unanswered flag is set to true.  Do this in a utils function.  If this is the case, direct to servicerating part of public app.  
Use namespace state_servicerating_questionX, where X is 1,2,3...
Include a q_id (question id) variable to each question state, as well as a variable with the question text.
For testing, also logically separate servicerating tests

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review

File: src/ussd_public.js [hunk 1]
```diff
@@ -651,6 +661,102 @@ go.app = function() {
             });
         });
 
+    // REGISTRATION STATES
+
+        // ChoiceState
+        self.add('state_servicerating_question1', function(name) {
+            //var q_id = '1';
+            var q_text_en = "Welcome. When you signed up, were staff at the facility friendly & helpful?";
```
Comment (imsickofmaps) [inline src/ussd_public.js]: put the `$()` around this text not the variable (and in all the other places)

File: test/ussd_public.test.js [hunk 2]
```diff
@@ -814,6 +814,148 @@ describe("familyconnect health worker app", function() {
             });
         });
 
+        // TEST SERVICERATING FLOW
+
+        describe("Servicerating testing", function() {
```
Comment (imsickofmaps) [inline test/ussd_public.test.js]: `Service Rating testing`

**Comment by imsickofmaps** [comment]
Minor things then :+1: when travis happy. Do we need to adjust anything on the session restart handling though?

**Comment by NicolaasDieBaas** [comment]
As part of the next issue (#40), I'll investigate whether anything needs changing on session restart.

**Pull request merged**